sphinxcontrib/openapi.py

```diff
@@ -122,10 +122,27 @@ def _httpresource(endpoint, method, properties):
     yield ''
 
 
-def openapi2httpdomain(spec):
+def _clean_paths_param(paths):
+    """ Clean paths string param
+
+    - Split string on commas to get list of paths to filter by
+    - Return None if unable to parse the string or paths not provided
+
+    """
+    if paths is None:
+        return None
+    paths = [p.strip() for p in str(paths).split(',')]
+    return paths if len(paths) > 0 else None
+
+
+def openapi2httpdomain(spec, paths=None):
```

Comment:
Yeah, I like this better (also referring to your second comment below, as its related). It will be more extensible down the road if you want to continue adding more options. I'll play with it a bit.